docs/brisket.createserver.md

```diff
@@ -117,8 +117,7 @@ var brisketServer = Brisket.createServer({
     apiHost: 'http://localhost:4000',
     clientAppRequirePath: 'app/ClientApp',
     environmentConfig: {
-        clientAppUrl: '//www.myapp.com/application.js',
-        startClientAppAsync: true
```

Comment:
why do you remove this option? this change will remove backwards compatibility. we should keep the backward compatibility for now. when https://github.com/bloomberg/brisket/pull/234 goes through, the user will be able to make this decision on their own